test/boost/user_types_test.cc

```diff
@@ -101,7 +101,15 @@ SEASTAR_TEST_CASE(test_user_type) {
 }
 
 SEASTAR_TEST_CASE(test_invalid_user_type_statements) {
-    return do_with_cql_env_thread([] (cql_test_env& e) {
+    // The test creates ut4 with a lot of fields,
+    // this may take a while in debug builds,
+    // to avoid raft operation timeout set the threshold
+    // to some big value.
+    co_await utils::get_local_injector().enable_on_all("group0-raft-op-timeout-in-ms", false, {
+        {"value", "1000000000"}
```

Comment:
That's ~277 hours

Why not use something reasonable -- let's say 5 minutes? I.e. 300000 ms

Reply:
What's reasonable here? `arm/debug` is completely unpredictable and may take arbitrary amount of time. This test is not about timeouts, why care about that?

Reply:
On one hand arbitrary value is acceptable because there was no timeout before so the test would just hang forever if there was a problem. And then the jenkins job would timeout after 10 hours.

But why not improve now if we can? Instead of hanging for 10 hours if there's a problem, heat up the planet and keep paying $$$ to Mr Bezos (due to the running AWS instances where we run the test) we can timeout much earlier than that.

5 minutes should be enough even for arm/debug. Let's make it 10 minutes just in case. But let's keep it reasonable